examples/hello-cube.rs

```diff
@@ -128,7 +128,7 @@ impl harmony::AppState for AppState {
         // We can't render anything without a camera. Add one here.
         // Thankfully we have a method to help that makes it easy!
         let mut camera_data = CameraData::new_perspective(
-            70.0,
+            70f32.to_radians(),
```

Comment:
Might be better to do this inside of `new_perspective`? I think I'd like to keep everything in degrees since it's normally easier to read.

Reply:
Changed it